src/crosschain/swapExactIn/thorChainSwap/constants.ts

```
@@ -0,0 +1,91 @@
+import { ChainId } from '../../../constants'
+import { GAS_TOKEN, Token } from '../../../entities'
+import { ThorChainError } from '../../sdkError'
+import type { ThorChainDestination } from './types'
+
+
+// BTC destination — dispatched via toBtcSwap (NOT thorChainSwap).
```

ESLint

[error] Delete `⏎`

(prettier/prettier)